project/Common.scala

```diff
@@ -56,6 +56,7 @@ object Common {
 
   private lazy val playDir = Def.setting[String] {
     if (playVer.value startsWith "2.6") "play-2.6"
+    else if (playVer.value startsWith "2.7") "play-2.7"
```

Comment:
Is there any diff between 2.6/2.7, otherwise rather map 2.6+

Reply:
It is `case class Multipart.FileInfo` which has [a new field](https://www.playframework.com/documentation/2.7.x/Migration27#New-fields-and-methods-added-to-FilePart-and-FileInfo) that needs to be matched. The other was needed because I got a compiler error about the unused `doc` variable`:
```diff
--- play-2.6/modules/reactivemongo/MongoController.scala	2019-02-07 16:36:33.514737323 +0100
+++ play-2.7/modules/reactivemongo/MongoController.scala	2019-02-08 08:03:44.019824416 +0100
@@ -72,7 +72,7 @@
   implicit def readFileReads[Id <: JsValue](implicit r: Reads[Id]): Reads[JsReadFile[Id]] = new Reads[JsReadFile[Id]] {
     def reads(json: JsValue): JsResult[JsReadFile[Id]] = json match {
       case obj: JsObject => for {
-        doc <- BSONDocumentFormat.partialReads(obj)
+        _ <- BSONDocumentFormat.partialReads(obj)
         _id <- (obj \ "_id").validate[Id]
         ct <- readOpt[String](obj \ "contentType")
         fn <- (obj \ "filename").toOption.fold[JsResult[Option[String]]](
@@ -168,7 +168,7 @@
     implicit def ec: ExecutionContext = materializer.executionContext
 
     parse.multipartFormData {
-      case Multipart.FileInfo(partName, filename, contentType) =>
+      case Multipart.FileInfo(partName, filename, contentType, _) =>
         Accumulator.flatten(gfs.map { gridFS =>
           val fileRef = fileToSave(filename, contentType)
 
```

Reply:
But, it is really a lot duplicate code; could we somehow separate the MongoController for Play 2.6 / 2.7?!